gen3/bin/kube-setup-secrets.sh

```diff
@@ -200,7 +200,7 @@ if ! g3kubectl get configmaps/fence > /dev/null 2>&1; then
 fi
 
 # Generate private and public keys for each service listed (except fence)
-services_list=( "analysis" "amanuensis" )
+services_list=( "analysis" "amanuensis", "gearbox" )
```

Comment:
The service signing the request is not gearbox is gearbox-middleware. This affects the name of the key as well in the other files changed here. from gearbox-jwt-keys-g3auto to gearbox-middleware-jwt-keys-g3auto